crates/floresta-chain/src/pruned_utreexo/consensus.rs

```diff
@@ -174,30 +263,24 @@ impl Consensus {
                 }
                 .into());
             }
+
+            // Check if the transaction is not creating more coins than can possibly exist in the chain.
             if output_value > 21_000_000 * 100_000_000 {
```

Comment:
NIT: we have constants for those values, no?

Reply:
we have `coin_value`